src/clims/services/workflow/camunda/client.py

```diff
@@ -204,6 +217,14 @@ def is_relative(name):
                 call_activity.attrib[
                     "calledElement"] = called_process_id_fully_qualified
 
+        # Qualify all form keys
+        user_tasks = process.findall("bpmn:userTask", ns)
+        for user_task in user_tasks:
+            form_key = user_task.attrib.get("{http://camunda.org/schema/1.0/bpmn}formKey", None)
+            if form_key and is_relative(form_key):
+                form_key_qualified = "{}.{}".format(namespace, form_key)
+                user_task.attrib["{http://camunda.org/schema/1.0/bpmn}formKey"] = form_key_qualified
```

Comment:
It took me a while before I realized that this method mutates the input argument. It's probably necessary, athough this kind of mutation is nasty. It would be nice if this side effect was expressed or made clear in some way.

Reply:
Ahh. The name of the method is `refine_xml` so I would say that it indicates that it's modifying it. Tell me if you think it requires more than that though!

Reply:
Yes, I know that the name indicates it, but I think in general that modifying an input argument is the least obvious way to mutate data. I'm not strongly against this method above, but I still wanted to point out my opinion about it :)

Reply:
Right, I get your point and I agree with it totally.